content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -120,10 +120,16 @@ The Organization Resource supports proprietary codes for:
 The Account Resource supports proprietary codes for:
 
 * Account.type.text codes are maintained in [Code Set 18736 Account Types](#code-set-18736-account-types)
-* Account.balance.currency codes are maintained in [Code Set 18934 Currency Type](#code-set-18934-currency-type)
-* Account.state codes are maintained in [Code Set 24451 Benefit Order Status](#code-set-24451-benefit-order-status) and [Code Set 4002640 Correspondence Status](#code-set-4002640-correspondence-status)
+* The Account Balance extensions on Account are maintained in [Code Set 18934 Currency Type](#code-set-18934-currency-type)
+* The Account State extensions on Account are maintained in [Code Set 24451 Benefit Order Status](#code-set-24451-benefit-order-status) and [Code Set 4002640 Correspondence Status](#code-set-4002640-correspondence-status)
```

Comment:
What's the reason for updating this on a ChargeItem PR?

Reply:
updated based on this comment https://github.com/cerner/fhir.cerner.com/pull/629#discussion_r623415995
updated description of PR.

Reply:
I asked for change for consistency of representing codes for extensions.